Add Heliostation (Working title)

remove32 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Add Heliostation, a map designed by Helianthus.

This map is larger than box and should be able to better support our average population.
The hallways are spacious and it should be easy to navigate to each area.
![heliostationmap](https://user-images.githubusercontent.com/27014495/82127788-d21b5680-977b-11ea-8873-85f04337a970.png)

Feedback thread: https://fulpstation.com/showthread.php?tid=1470

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Primarily we want to have our own map - this is the first step towards accomplishing that. 
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: New map - Heliostation!
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Commit:
Revert "testing git"

This reverts commit fa5d6d86a73462dd94629a125d9112df34ef2d6f.

Commit:
testing git

hi

Commit:
add newstation

thanks Helianthus#9774 !

Helianthus00 committed:
Update newstation.dmm

Commit:
move captain

Helianthus00 committed:
Update newstation.dmm

Changed atmos to look better and finished repiping station.

Helianthus00 committed:
Update newstation.dmm

added gas meters (need to check layer 1 pipes to see if they still work)

Helianthus00 committed:
Update newstation.dmm

Moved the turbine into the atmos room. Left the old one too, just in case.

Helianthus00 committed:
Update newstation.dmm

Removed atmos turbine and added doors to hydroponics

Helianthus00 committed:
Update newstation.dmm

chapel, space burial, vacant office, maint atmos, and grav gen relocated. Art closet and emergency closet displaced temporarily.

Helianthus00 committed:
Update newstation.dmm

Helianthus00 committed:
Update newstation.dmm

small changes

Helianthus00 committed:
Update newstation.dmm

minor changes (again)

Helianthus00 committed:
Update newstation.dmm

big engineering changes. Reworked AI sat atmos.

Helianthus00 committed:
Update newstation.dmm

minor engineering pipe changes, moved engineering pod

Helianthus00 committed:
Update newstation.dmm

Helianthus00 committed:
Update newstation.dmm

minor changes + plushes

Helianthus00 committed:
Update newstation.dmm

added event spawners and minor tweaks.

Helianthus00 committed:
Update newstation.dmm

All areas limited to one air alarm, except the engine. Minor pipe changes and other tweaks.

Helianthus00 committed:
Update newstation.dmm

LOTS of small bug changes, fixes, etc.

Helianthus00 committed:
Update newstation.dmm

area changes, added some cameras

Helianthus00 committed:
Update newstation.dmm

added another important camera.

Helianthus00 committed:
Update newstation.dmm

Filled in some empty maint space

Helianthus00 committed:
Update newstation.dmm

xeno cold room aesthetic changes

Helianthus00 committed:
Update newstation.dmm

AI sat minor changes

Helianthus00 committed:
Update newstation.dmm

more scrubbers, more maint, maint doors, idk a lot of stuff.

Helianthus00 committed:
Update newstation.dmm

Brand, spanking new AI sat. New security reception. Other small changes.

Old AI sat is shifted away just in case.

Helianthus00 committed:
Update newstation.dmm

rad room added for gravity gen. Small cargo maint changes

Helianthus00 committed:
Update newstation.dmm

small fixes, removed old AI sat

Helianthus00 committed:
Update newstation.dmm

more small changes

Helianthus00 committed:
Update newstation.dmm

various small fixes, apc areastring fixes

Helianthus00 committed:
Update newstation.dmm

lowered medbay backup SMES charge, as well as the solars' SMESs, aside from security, since it is such a huge power drain.

Helianthus00 committed:
Update newstation.dmm

bug fixes

Helianthus00 committed:
Update newstation.dmm

lots of big changes in security, surgery B, other small stuff.

Helianthus00 committed:
Update newstation.dmm

cap's quarters and tele room swap.

Helianthus00 committed:
Update newstation.dmm

security med rooms changed up a bit, added a windoor to the brig one.

Helianthus00 committed:
Update newstation.dmm

kitchen changes, added op table to morgue, changed sec door access

Helianthus00 committed:
Update newstation.dmm

fixes

Helianthus00 committed:
Update newstation.dmm

more stamps

Helianthus00 committed:
Update newstation.dmm

more security changes. Added a cell, increased holding cell size.

Helianthus00 committed:
Update newstation.dmm

flash added to armory

Helianthus00 committed:
Update newstation.dmm

armory window

Helianthus00 committed:
Update newstation.dmm

plasmeme medbay added

Helianthus00 committed:
Update newstation.dmm

minor changes.

Helianthus00 committed:
Update newstation.dmm

more plasmamed changes

Helianthus00 committed:
Update newstation.dmm

fixed plasmamed door ID

Helianthus00 committed:
Update newstation.dmm

added notice boards

Helianthus00 committed:
Update newstation.dmm

lots and lots of small changes

Helianthus00 committed:
Update newstation.dmm

chairs added to genetics

remove32 committed:
Merge branch 'master' of https://github.com/TheSwain/Fulpstation

Helianthus00 committed:
Update newstation.dmm

very minor updates

Helianthus00 committed:
Update newstation.dmm

minor fixes

Helianthus00 committed:
Update newstation.dmm

more cables to prevent half the station losing power from a mouse

Helianthus00 committed:
Update newstation.dmm

fixed kitchen table being inaccessible

Helianthus00 committed:
Update newstation.dmm

small fixes

Helianthus00 committed:
Update newstation.dmm

moved a lot of disposals pipes in order to fix the layer1 pipe sprite bug.

Helianthus00 committed:
Create emergency_news.dmm

adding the new escape shuttle.

Helianthus00 committed:
Create floor.dm

new floor code, for brown shuttle tile

Helianthus00 committed:
Revert "Create floor.dm"

This reverts commit aed94e2c40ba79854f8e3ed2fdeefffe96c4a0da.

Helianthus00 committed:
Create floors.dmi

new brown shuttle tile

Helianthus00 committed:
Delete floors.dmi

file was in the wrong place. Whoops

Helianthus00 committed:
Update floors.dmi

Helianthus00 committed:
Update mineral_floor.dm

code to add brown titanium floors

Helianthus00 committed:
Update newstation.dmm

fixed broken disposals in det's office

Helianthus00 committed:
Update newstation.dmm

incorrect sortType for chaplain

Helianthus00 committed:
Update newstation.dmm

fixed more sortTypes

Helianthus00 committed:
Update newstation.dmm

lights added to tele room and cap's quarters

Helianthus00 committed:
Update newstation.dmm

fixed cargo conveyors

Helianthus00 committed:
Update newstation.dmm

minor changes to the external engine

Helianthus00 committed:
Update newstation.dmm

added some lights

Helianthus00 committed:
Update newstation.dmm

incinerator area fix to power outside pump

Helianthus00 committed:
Update newstation.dmm

incinerator gas filter fixed

Helianthus00 committed:
Update newstation.dmm

added a seperate air alarm for the incinerator chamber, similar to how the sm engine works.

Helianthus00 committed:
Create heliostation.dmm

Changed name, added janitor lizard

Helianthus00 committed:
Create heliostation.json

fixed json file

Helianthus00 committed:
Delete newstation.dmm

removed old station

Helianthus00 committed:
Delete newstation.json

removed old json

Helianthus00 committed:
Update heliostation.dmm

fixed some pipe and cable connections in med

Helianthus00 committed:
Update heliostation.dmm

added a bunch of scrubbers/vents, more sandbags in mining lockers/bay

Helianthus00 committed:
Update heliostation.dmm

wayfinding beacons added

Helianthus00 committed:
Update heliostation.dmm

added bot decals to delivery areas

Helianthus00 committed:
Update heliostation.dmm

Helianthus00 committed:
Update heliostation.dmm

windoors at botany

Helianthus00 committed:
Update heliostation.dmm

NEW new botany

EgorDinamit commented on _maps/heliostation.json:
```diff
@@ -0,0 +1,11 @@
+	    "cargo": "cargo_box",
+	    "ferry": "ferry_fancy",
+	    "whiteship": "whiteship_box",
+	    "emergency": "emergency_box"
```
```suggestion
	    "emergency": "emergency_news"
```

Homingpenguins replied:
is that all i had to do to add that...

remove32 committed:
Update _maps/heliostation.json

Co-authored-by: EgorDinamit <53223414+EgorDinamit@users.noreply.github.com>

Helianthus00 committed:
Update heliostation.dmm

lots of small changes, every suggestion up to aaaa1023's post on the forums, and a few other things I noticed. Fixed engineering and cargo bay access.

Helianthus00 committed:
Update maps.txt

changed the map.txt to reflect the new map name.

Helianthus00 committed:
Update heliostation.json

shuttle's BROKE, fixing json

Helianthus00 committed:
Update heliostation.dmm

a few more small changes based on map errors.

Helianthus00 committed:
Update heliostation.dmm

more secure AI upload

Helianthus00 committed:
Update heliostation.dmm

renamed research windoor

Helianthus00 committed:
Update heliostation.dmm

another small fix, changed sec lockers

Helianthus00 committed:
Update heliostation.dmm

gives chef access to botany

Helianthus00 committed:
Update heliostation.dmm

fixed z-level bug. Added maint TEG, a couple other small changes

Helianthus00 committed:
Update heliostation.dmm

surrounded the AI solars with reinforced walls. It was just too penetrable.

Helianthus00 committed:
Update heliostation.dmm

metal for chem lab and a bunch more emclosets and fire closets

Helianthus00 committed:
Update heliostation.dmm

welding masks are no longer collectable.

Helianthus00 committed:
Update heliostation.dmm

AI upload cams readded

Helianthus00 committed:
Update heliostation.dmm

changed prison APC to highcap, Added a few holopads, flipped morgue thing in maint aux room, other small stuff.

Helianthus00 committed:
Update heliostation.dmm

fixed the cryo freezer

Helianthus00 committed:
Update heliostation.dmm

added gen maint access to the morgue.

Helianthus00 committed:
Update heliostation.dmm

flipped the ORM again and added a few more spawnpoints.

Helianthus00 committed:
Update heliostation.dmm

added department signs

Helianthus00 committed:
Update heliostation.dmm

flipped the ORM again, turns out it was right before, whoops.

Helianthus00 committed:
Update heliostation.dmm

fixed the sloppy floor paint at cargo

Helianthus00 committed:
Update heliostation.dmm

changed cargo wood amount.

Helianthus00 committed:
Update heliostation.dmm

added conveyors to cargo warehouse

Helianthus00 committed:
Update heliostation.dmm

added nanite shutter button in RD office, as well as a blast protection poddoor to his left window.

Helianthus00 committed:
Update heliostation.dmm

fixed bar air alarm

Helianthus00 committed:
Update heliostation.dmm

fixed an APC in the AI sat

Helianthus00 committed:
Update heliostation.dmm

upped the layer on the holding cell flasher.

Helianthus00 committed:
Update heliostation.dmm

added a bunch of holopads, as well as fixing a scrubber pipe in perma

Helianthus00 committed:
Update heliostation.dmm

changed HoP fire alarm

Helianthus00 committed:
Update heliostation.dmm

fixed chamber air alarm

Helianthus00 committed:
Update heliostation.dmm

added shutters and glass doors to prisoner cells

Helianthus00 committed:
Update heliostation.dmm

added more atmos access in engineering

Helianthus00 committed:
Update heliostation.dmm

Blocked in AI solars causes issues with the solar plates.

Helianthus00 committed:
Update heliostation.dmm

removed the extra holopad in tech storage and fixed the one scrubber/vent in the hallway is normalized

Helianthus00 committed:
Update heliostation.dmm

added cameras to each prison cell

Helianthus00 committed:
Update heliostation.dmm

moved the coffin storage door to line up with the glass

Helianthus00 committed:
Update heliostation.dmm

added an aux docking port

Helianthus00 committed:
Update heliostation.dmm

fixed areas

Helianthus00 committed:
Update heliostation.dmm

fixed brig medical door

Helianthus00 committed:
Update heliostation.dmm

small fixes to lights

Samuuu3 commented:
I really love this map.

Helianthus00 committed:
Update heliostation.dmm

fixed a small mistake with the engineering entrance

Helianthus00 committed:
Update heliostation.dmm

fixed genetics doors (maybe)

Helianthus00 committed:
Update heliostation.dmm

fixed the SM being on the main power line. Rewired the APC to give it power from the SMES roundstart as well.

Helianthus00 committed:
Update heliostation.dmm

fixed engineering access.
